src/toniarts/openkeeper/tools/convert/AssetsConverter.java

```diff
@@ -142,11 +143,12 @@ public String toString() {
     public static final String MATERIALS_FOLDER = "Materials";
     public static final String MODELS_FOLDER = "Models";
     public static final String TEXTURES_FOLDER = "Textures";
-    public static final String MAP_THUMBNAILS_FOLDER = TEXTURES_FOLDER + File.separator + "Thumbnails";
-    public static final String MOUSE_CURSORS_FOLDER = "Interface" + File.separator + "Cursors";
-    public static final String FONTS_FOLDER = "Interface" + File.separator + "Fonts";
-    public static final String TEXTS_FOLDER = "Interface" + File.separator + "Texts";
-    public static final String PATHS_FOLDER = "Interface" + File.separator + "Paths";
+    public static final String MAP_THUMBNAILS_FOLDER = "Thumbnails";
+    public static final String INTERFACE_FOLDER = "Interface" + File.separator;
```

Comment:
A small thing, but since this folder holds no significance (?, only used internally), change to private modifier? :) Very small thing...